metabase/metabase
# Add remove columns panel

Closes https://github.com/metabase/metabase/issues/33841

### Description
This PR adds a new panel to control adding and removing table columns from a query, separating the idea of add/remove from show/hide in viz settings. The new panel is only available in structured queries (and should not be displayed for dashcards), and should allow you to implicitly add fields from related tables, search available columns, and perform bulk add / remove column operations per table.

### How to verify
Create a structured question and try adding / removing columns from a query. A few points:
- Showing / Hiding should not re-run queries
- Try using saved questions as table sources
- You should now be able to hide columns from the table header


### Checklist
